.travis.yml

```diff
@@ -23,7 +23,7 @@ before_script:
   - docker-compose up -d
   - bash -c 'while [[ "$(curl --insecure -s -o /dev/null -w ''%{http_code}'' http://localhost:8000/healthcheck)" != "200" ]]; do sleep 10 && docker logs cccatalog-api_web_1; done'
   - ./load_sample_data.sh
-  - bash -c 'while [[ "$(curl --insecure -s -o /dev/null -w ''%{http_code}'' http://localhost:8000/image/search?q=test)" != "200" ]]; do sleep 5; done'
+  - bash -c 'while [[ "$(curl --insecure -s -o /dev/null -w ''%{http_code}'' http://localhost:8000/v1/images?q=test)" != "200" ]]; do sleep 5; done'
```

Comment:
Backwards compatibility is implemented at the infrastructure level, so it would take some hackery to get it to work on the legacy API. Since the legacy API is frozen at an old commit, there isn't a need to continue testing it.